packages/openid4vc-client/src/OpenId4VcClientService.ts

```diff
@@ -147,18 +147,26 @@ export class OpenId4VcClientService {
     }
   }
 
-  private assertCredentialHasFormat(format: string, scope: string, metadata: EndpointMetadata) {
-    if (!metadata.openid4vci_metadata) {
+  private assertCredentialHasFormat(supportedFormats: Array<string>, scope: string, metadata: EndpointMetadata) {
```

Comment:
Not needed for draft 11 support, but with the JWT credentials coming up, I assume we want to support them as well.